www/src/scss/themes/helpers/_theme-generator.scss

```diff
@@ -45,6 +45,12 @@
         );
       }
     }
+
+    &.larivaar.larivaar-assist {
```

Comment:
so larivaar assist will have the highest priority? would some messaging help so users don't get confused?